app/init.php

```diff
@@ -724,6 +724,25 @@ function (mixed $value) {
     $providerName = System::getEnv('_APP_LOGGING_PROVIDER', '');
     $providerConfig = System::getEnv('_APP_LOGGING_CONFIG', '');
 
+    try {
+        $loggingProvider = new DSN($providerConfig ?? '');
```

Comment:
lets also check if we initalise the logger resource in worker.php , cli.php, realtime.php and any other places.